<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>agentsearch</class>
 <widget class="QMainWindow" name="agentsearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agent Details</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="F1">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>1368</width>
      <height>878</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1368</width>
      <height>878</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1368</width>
      <height>878</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(169, 173, 255);
border-radius:20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="F2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>821</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 222, 210);
border-radius:20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="ACCT">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AGENT ID</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="mob">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>80</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MOBILE NO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b1">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>190</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(198, 198, 198);</string>
     </property>
     <property name="text">
      <string>SEARCH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>310</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PHONE NO</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ADDRESS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>540</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>APPOINTED
 BANK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>590</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GENDER:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>480</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>APPOINTED
 CUSTOMER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(194, 247, 255);</string>
     </property>
     <property name="text">
      <string>SEARCH TYPE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>130</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(194, 247, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Enter Search Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="nam">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>250</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ph">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255)
;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="em">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>360</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="add">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>420</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="appC">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>480</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(194, 247, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="appB">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>530</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>10</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255, 221, 255)</string>
     </property>
     <property name="text">
      <string>SEARCH AGENT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gen">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>590</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(194, 247, 255)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1352</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
